src/redux/actions/actionTypes.js

```diff
@@ -8,6 +8,22 @@ export const LOGIN_START = 'LOGIN_START';
 export const LOGIN_SUCCESS = "LOGIN_SUCCESS";
 export const LOGIN_FAILURE = "LOGIN_FAILURE";
 
+export const LOGOUT = 'LOGOUT';
+
+//Move players
+// export const MOVE_PLAYER_UP = 'MOVE_PLAYER_UP';
+// export const MOVE_PLAYER_DOWN = 'MOVE_PLAYER_DOWN';
+// export const MOVE_PLAYER_RIGHT = 'MOVE_PLAYER_RIGHT';
+// export const MOVE_PLAYER_LEFT = 'MOVE_PLAYER_LEFT';
+export const MOVE_PLAYER_ERROR = 'MOVE_PLAYER_ERROR';
+
+export const MOVE_PLAYER = 'MOVE_PLAYER';
+
+//ROOMS/GRID
+export const SAVE_ROOMS = 'SAVE_ROOMS';
```

Comment:
I will most likely overwrite this, too. But leave it in and I'll worry about that. :)